include/NAS2D/MathUtils.h

```diff
@@ -38,17 +38,17 @@ namespace NAS2D
  * Applies a linear conversion to map an input domain to an output range.
  *
  * \param	value: The value to be converted from the input domain to the output range.
- * \param	domainPoint1: A known lower fixed point in the domain. Must be less than domainPoint2.
- * \param	domainPoint2: A known upper fixed point in the domain. Must be greater than domainPoint1.
- * \param	rangePoint1: A known lower fixed point in the range. Must be less than or equal to rangePoint2.
- * \param	rangePoint2: A known upper fixed point in the range. Must be greater than or equal to rangePoint1.
+ * \param	domainPoint1: A known fixed point in the domain.
+ * \param	domainPoint2: Another known fixed point in the domain.
+ * \param	rangePoint1: A known fixed point in the range.
+ * \param	rangePoint2: Another known fixed point in the range.
```

Comment:
What if we throw in the word "corresponding" here:
"corresponding known fixed point"